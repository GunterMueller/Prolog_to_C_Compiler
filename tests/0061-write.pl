main :-
	write(123), nl,
	write([]), nl,
	write('abc def'), nl,
	write([1,2]), nl,
	write([1,2|3]), nl,
	write((foo(X, Y) :- bar, !, baz(X))), nl,
	write((1 + 2) / 3), nl,
	write(1 - -2), nl,
	writeq(123), nl,
	writeq([]), nl,
	writeq('abc def'), nl,
	writeq([1,2]), nl,
	writeq([1,2|3]), nl,
	writeq((foo(X, Y) :- bar, !, baz(X))), nl,
	writeq((1 + 2) / 3), nl,
	writeq(1 - -2), nl.
