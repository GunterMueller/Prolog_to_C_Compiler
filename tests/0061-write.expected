123
[]
abc def
[1,2]
[1,2|3]
foo(_40,_41) :- bar,!,baz(_40)
(1+2)/3
1- -2
123
[]
'abc def'
[1,2]
[1,2|3]
foo(_40,_41) :- bar,!,baz(_40)
(1+2)/3
1- -2
